spring-xd-shell/src/test/java/org/springframework/xd/shell/AbstractShellIntegrationTest.java

```diff
@@ -89,8 +90,11 @@
 	@BeforeClass
 	public static synchronized void startUp() throws InterruptedException, IOException {
 		RandomConfigurationSupport randomConfigSupport = new RandomConfigurationSupport();
+		// todo: Ability to verify if the transport is valid and the required message broker is setup.
+		String transport = System.getProperty("XD_TRANSPORT");
```

Comment:
This feels wrong:
1. either we set things in stone (like it was before)
2. or we allow selecting a transport to be selected with a prop, but then the defaults are coded at two places: here and in SingleNodeApplication itself